charts/lenses/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 description: A chart for Lenses
 name: lenses
-version: 2.2.14
-appVersion: 2.2.9
+version: 2.2.15
+appVersion: 2.2.10
```

Comment:
We don't need to bump the appVersion as we 2.2.9 is latest Lenses release

Reply:
Ok, i was unsure if i needed to change that. I fix.

Reply:
hm doesnt seem to work on linux.
```
nic@ucnd8132rff:~/development/kafka-helm-charts$ ./package.sh 
Cleaning and preparing
rm: missing operand
Try 'rm --help' for more information.
```